src/test/java/com/agilethought/atsceapi/ATSceApiApplicationTests.java

```diff
@@ -1,6 +1,7 @@
 package com.agilethought.atsceapi;
 
 import org.junit.jupiter.api.Test;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
```

Comment:
Why did you add this? I believe this is also outside of the scope of your tasks, I think it's better if you delete it, this should also not appear in your commit